test/mocha/release.js

```diff
@@ -8,7 +8,7 @@ function run(command, args, done) {
         process.stdout.write("\0");
     }, 5 * 60 * 1000);
     spawn(command, args, {
-        stdio: "ignore"
+        stdio: [ "ignore", 1, 2 ]
```

Comment:
FYI, these extra lines are the result of the aforementioned change:

https://travis-ci.org/mishoo/UglifyJS2/jobs/242117329#L1299-L1402